app/org/sagebionetworks/bridge/play/controllers/TaskController.java

```diff
@@ -30,14 +34,28 @@ public void setTaskService(TaskService taskService) {
         this.taskService = taskService;
     }
     
-    public Result getTasks(String untilString) throws Exception {
+    public Result getTasks(String untilString, String offset, String daysAhead) throws Exception {
```

Comment:
Would it be useful to take "now" as part of the API, so app developers can test the schedule on the client side without mucking with the server side? If the "now" parameter isn't passed in, we could default to the true "now".